### ISSUE TITLE
[Suggestion/Question] Is it possible to add a config to enable/disable certain structures (ingame or JSON)? (For fabric 1.20.1)

### ISSUE THREAD

**Issue by TheShadyyOneYT** [frqnny/mostructures#78]
I ask this because I don't feel like the Hot air balloon, and the lighthouse will fit in with my modpack I'm making as it has a medieval ring to it. It would be lovely to disable certain structures from spawning. I'm no expert in coding in any way but maybe like something `air_baloon_structure_spawns=true` or something like that. Once again, I'm no expert in coding, and I barely understand datapacks and modding as is. But something feasible to use would be great, as in the mean time I can't use this mod for my modpack until I can disable these structures, thank you for reading this. ~Cheers, TheShadyy

**TL;DR: Config to enable/disable certain structures via JSON, or TOML, or ingame.**

**Comment by frqnny** [comment]
The config does that already
